File: C#/CSharpDevelopmentContainerAlpha/CSharpDevelopmentContainerAlpha/Constructor test/Petrik.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Constructor_test
{
	class Petrik : Clovek
	{
		public Petrik(string meno, int vek) : base(meno, vek) { }
		public Petrik(string meno, int vek, string zamestnanie) : base(meno, vek, zamestnanie) { }

		public override void Pozdrav()
		{

			base.Pozdrav();
			Console.WriteLine("xd Petrik !");

		}

	}
}

File: Java/eclipse2017/DevelopmentContainerA/src/dedenietest/Petrik.java
package dedenietest;

public class Petrik extends Clovek {
	
	public Petrik(String meno, int vek) {
		
		super(meno, vek);
		
	}
	
	
	@Override
	public void pozdrav() {
		
		super.pozdrav();
		System.out.println("A SOM PETRIK !!");
		
	}
	

}
